tests/test_encoder.py

```diff
@@ -241,3 +241,37 @@ def test_encode_rounding_floats(self):
             input_geometry=geometry,
             expected_geometry=exp_geoemtry,
         )
+
+
+class QuantizeTest(unittest.TestCase):
+
+    def test_quantize(self):
+        from mapbox_vector_tile import decode
+        from mapbox_vector_tile import encode
+        props = dict(foo='bar')
+        shape = 'POINT(15 15)'
+        feature = dict(geometry=shape, properties=props)
+        features = [feature]
+        source = dict(name='layername', features=features)
+        bounds = 10.0, 10.0, 20.0, 20.0
+        pbf = encode(source, quantize_bounds=bounds)
+        result = decode(pbf)
+        act_feature = result['layername']['features'][0]
+        act_geom = act_feature['geometry']
+        exp_geom = [[2048, 2048]]
+        self.assertEqual(exp_geom, act_geom)
```

Comment:
what about a combinaison of `quantize_bounds` and `y_coord_down` ?

And use a point that is not excatly in the middle of the bounds to test the impact of `y_coord_down` parameter